.github/README.md

```diff
@@ -0,0 +1,108 @@
+# E2E Tests on GitHub CI
+
+E2E tests are executed on GitHub CI using the [E2E Tests Reusable Workflow](#about-the-reusable-workflow) or dedicated workflows.
+
+## Automatic workflows
+
+These workflows are designed to run automatically at crucial stages of the software development process, such as on every commit in a PR, nightly or before release.
+
+### PR E2E Tests
+
+Run on every commit in a PR to ensure changes do not introduce regressions.
+
+Link: https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/integration-tests.yml
+
+### Nightly E2E Tests
+
+Conducted nightly to catch issues that may develop over time or with accumulated changes.
+
+Link: https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/run-nightly-e2e-tests.yml
+
+### Release E2E Tests
+
+Currently, there are no automatic workflows triggering E2E tests at release. These tests are triggered on-demand by QA as needed.
+
+## On-Demand Workflows
+
+Triggered manually by QA for specific testing needs.
+
+**Examples:**
+
+- [On-Demand Automation Tests](https://github.com/smartcontractkit/chainlink/actions/workflows/automation-ondemand-tests.yml)
+- [CCIP Chaos Tests](https://github.com/smartcontractkit/chainlink/actions/workflows/ccip-chaos-tests.yml)
+- [OCR Soak Tests](https://github.com/smartcontractkit/chainlink/actions/workflows/on-demand-ocr-soak-test.yml)
+- [VRFv2Plus Smoke Tests](https://github.com/smartcontractkit/chainlink/actions/workflows/on-demand-vrfv2plus-smoke-tests.yml)
+
+## Test Configs
+
+E2E tests utilize TOML files to define their parameters. Each test is equipped with a default TOML config, which can be overridden by specifying an alternative TOML config. This allows for running tests with varied parameters, such as on a non-default blockchain network. For tests executed on GitHub CI, both the default configs and any override configs must reside within the git repository. The `test_config_override_path` workflow input is used to provide a path to an override config.
```

Comment:
maybe say also that this override config doesn't entirely override the original one? i.e. it can be partial, for example override just a single value

Reply:
@Tofel I can add a reference to the test config docs in CTF, wdyt?

Reply:
added reference to test config in CTF